### ISSUE TITLE
BuddyPress Docs integration

### ISSUE THREAD

**Issue by boonebgorges** [boonebgorges/participad#3]

**Comment by JohnMcLear** [comment]
@boonebgorges I assume these are docs for how to integrate with BuddyPress?

**Comment by boonebgorges** [comment]
No, it's a plugin that manages collaborative editing in the context of
BuddyPress groups and profiles.
http://github.com/boonebgorges/buddypress-docs
